core/src/test/java/org/infinispan/persistence/LocalModePassivationTest.java

```diff
@@ -59,31 +55,25 @@ protected void configureConfiguration(ConfigurationBuilder cb) {
 
    @Override
    protected EmbeddedCacheManager createCacheManager() throws Exception {
-      cacheStoreDir = new File(TestingUtil.tmpDirectory(this.getClass()));
-      Util.recursiveFileRemove(cacheStoreDir);
-
       ConfigurationBuilder builder = getDefaultClusteredCacheConfig(CacheMode.LOCAL, true, true);
       builder.transaction().transactionMode(TransactionMode.TRANSACTIONAL).lockingMode(LockingMode.PESSIMISTIC)
             .transactionManagerLookup(new EmbeddedTransactionManagerLookup())
-            .eviction().maxEntries(150).strategy(EvictionStrategy.LIRS)
+            .memory().storageType(StorageType.BINARY).size(150)
```

Comment:
That is just what the test was doing before :)  All I did was update it to use the new element.